requirements/base.in

```diff
@@ -2,46 +2,49 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.29.0
+wheel
 
-django==1.9.4
+django
```

Comment:
Personally leaving all the top level requirements undefined feels odd and a bit dangerous. But I guess this works too, instead of bumping versions directly here one needs to start locking them whenever some upgrade doesn't fit in - which will happen sooner rather than later.

Your call, if no one else chips in :+1: